Framework/Algorithms/inc/MantidAlgorithms/GeometryInfo.h

```diff
@@ -0,0 +1,39 @@
+#ifndef MANTID_ALGORITHMS_GEOMETRYINFO_H_
+#define MANTID_ALGORITHMS_GEOMETRYINFO_H_
+
+#include "MantidAlgorithms/BasicInstrumentInfo.h"
+
+namespace Mantid {
+
+namespace Geometry {
+class Instrument;
+class IDetector;
+}
+
+namespace API {
+class ISpectrum;
+}
+
+namespace Algorithms {
+
+class DLLExport GeometryInfo {
```

Comment:
Not really the point of this PR, but I just spotted this line with DLLExport. I think we have plenty of bad examples where we use the generic DLLExport define where I think we should be using the library specific defines (MANTID_ALGORITHMS_DLL here, from the algorithms DllConfig.h.
Not sure if there's something special about algorithms. We only have one that uses MANTID_ALGORITHMS_DLL: ExtractUnmaskedSpectra!.